buigiaanfb1/extract-data
# [US11][An] Uplift guidelines UI

## ⚡️ What problem does this PR solve
Uplift guidelines UI

<Explain by answer the question above>

## 🏗 How does this PR solve it

Change component History to Guidelines

## ✅ How can reviewers test the changes

1. Clone the project and take a look at the tree directory
3. Run `npm run dev` at the root directory
4. See left navigation with the `Guidelines` route
## 🖼 Screenshots or GIFs
<img width="1280" alt="Screen Shot 2022-10-12 at 16 51 48" src="https://user-images.githubusercontent.com/36767199/195311361-c726ddd0-b42a-4942-82ab-fbdb413623d3.png">

## 👌 Reviewer Checklist

- It meets the acceptance criteria
- Coding style is adhered to
- Architecture is appropriate
